keyboards/nuphy/air75_v2/ansi/user_kb.c

```diff
@@ -0,0 +1,419 @@
+/*
+Copyright 2023 @ Nuphy <https://nuphy.com/>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "user_kb.h"
+#include "ansi.h"
+#include "usb_main.h"
+#include "rf_driver.h"
+
+kb_config_t     kb_config;
+DEV_INFO_STRUCT dev_info = {
+    .rf_baterry = 100,
+    .link_mode  = LINK_USB,
+    .rf_state   = RF_IDLE,
+};
+bool f_bat_hold         = 0;
+bool f_sys_show         = 0;
+bool f_sleep_show       = 0;
+bool f_send_channel     = 0;
+bool f_dial_sw_init_ok  = 0;
+bool f_rf_sw_press      = 0;
+bool f_dev_reset_press  = 0;
+bool f_rgb_test_press   = 0;
+bool f_bat_num_show     = 0;
+
+uint8_t        rf_blink_cnt          = 0;
+uint8_t        rf_sw_temp            = 0;
+uint8_t        host_mode             = 0;
+uint16_t       rf_linking_time       = 0;
+uint16_t       rf_link_show_time     = 0;
+uint16_t       no_act_time           = 0;
+uint16_t       dev_reset_press_delay = 0;
+uint16_t       rf_sw_press_delay     = 0;
+uint16_t       rgb_test_press_delay  = 0;
+host_driver_t *m_host_driver         = 0;
+
+extern bool               f_rf_new_adv_ok;
+extern report_keyboard_t *keyboard_report;
+extern report_nkro_t *nkro_report;
+extern uint8_t            bitkb_report_buf[32];
+extern uint8_t            bytekb_report_buf[8];
+extern uint8_t            side_mode;
+extern uint8_t            side_light;
+extern uint8_t            side_speed;
+extern uint8_t            side_rgb;
+extern uint8_t            side_colour;
+
+/**
+ * @brief  gpio initial.
```

Comment:
`init` here stands for “initialize”. And capitalize [GPIO](https://en.wikipedia.org/wiki/General-purpose_input/output) because it’s an acronym.

```suggestion
 * @brief  Initialize GPIO.
```